cpp/modmesh/math/Complex.hpp

```diff
@@ -111,6 +140,29 @@ struct ComplexImpl
 template <typename T>
 using Complex = detail::ComplexImpl<T>;
 
+template <typename T>
+bool operator<(const Complex<T> & lhs, const Complex<T> & rhs)
+{
+    return lhs.norm() < rhs.norm();
+}
+
+template <typename T>
+bool operator>(const Complex<T> & lhs, const Complex<T> & rhs)
+{
+    return lhs.norm() > rhs.norm();
+}
```

Comment:
These comparison operator would be used in `SimpleArray::min()`, `SimpleArray::max()`.

Reply:
I think this is worth being a code comment.